### DISCUSSION TITLE
Is there possibility to add user_id to the short_urls table and populate that if user is authorized?

### DISCUSSION THREAD

**Discussion by thecodeholic** [ash-jc-allen/short-url#165] [Q&A]
Thank you for creating such a nice package. I really like it. 
Appreciate your hard work!

I want to associate every short URL to user when the user is authorized. Is there possibility to add user_id to the short_urls table and populate that if user is authorized?
Adding a column is easy, I can create new migration and run that. The problem is populating that new field.
I searched in existing closed issued, but could not found such feature. 
I also had a look at the code and found that there is no possibility to "inject" any kind of new data into short_urls table.

Am I missing something, or this feature is not available?

[Update]
The only solution I see at the moment is to get the object from `->make()` method and update it.

Cheers

**Comment by ash-jc-allen** [comment]
Hey @thecodeholic! Thanks, I'm really glad that you like the package so far.

At the moment, there isn't any functionality built into the package that allows you to add extra fields. As you say, you could update the `ShortURL` object after it's been created, but that's probably not ideal.

I was going to add this feature so that I could use it in a project that I was working on myself. But I didn't end up needing the feature, so I never got around to adding it.

For the time being, I think I'll hold off on adding a feature like this. But if enough people want it, I'll definitely consider adding it 😄
